compiler/rustc_session/src/config.rs

```diff
@@ -168,6 +168,8 @@ pub enum MirSpanview {
 pub enum InstrumentCoverage {
     /// Default `-C instrument-coverage` or `-C instrument-coverage=statement`
     All,
+    /// Additionally, instrument branches and output branch overage
+    Branch,
```

Comment:
To save anyone else the trouble of checking whether this is accidentally stable (it's not):

New values of `InstrumentCoverage` are automatically treated as nightly-only unless they are explicitly added to the check shown below on line 2674, so `Branch` is correctly treated as a nightly-only value. 👍 

https://github.com/rust-lang/rust/blob/c6f5495591f1688b7d47a3c4085d84bd7dedde4c/compiler/rustc_session/src/config.rs#L2665-L2685

Reply:
Also, I haven’t yet hooked up the option, so it doesn’t do anything right now, yet.